rasa/shared/core/training_data/story_reader/markdown_story_reader.py

```diff
@@ -354,9 +368,8 @@ def is_test_stories_file(file_path: Union[Text, Path]) -> bool:
         if not rasa.shared.data.is_likely_markdown_file(file_path):
             return False
 
-        dirname = os.path.dirname(file_path)
         return (
-            DEFAULT_E2E_TESTS_PATH in dirname
+            MarkdownStoryReader._is_tests_file(file_path)
```

Comment:
optional: I'm a bit torn here. In my opinion this is rather an`improvement` than a `bug` and as markdown is deprecated I wouldn't change any of its behavior unless it's a bug. So in my opinion file also to just keep the old behavior here. This might also reduce the effort in case we have to do a follow-up PR for some reason in these parts.